dbdialog/src/main/java/org/pentaho/ui/database/event/DbInfoHandler.java

```diff
@@ -0,0 +1,15 @@
+package org.pentaho.ui.database.event;
```

Comment:
Does kettle use a top-level license file? I didn't think it did. I think we need a copywrite statement in each source file.

Reply:
there is a top-level license file, at least for plugins that was considered enough now